ranger/gui/ui.py

```diff
@@ -485,6 +491,33 @@ def get_pager(self):
     def _get_viewmode(self):
         return self._viewmode
 
+    def _set_titlebar(self, value):
+        if isinstance(value, Signal):
+            value = value.value
+        if value == '':
+            value = self.ALLOWED_STATUS[0]
+        if value in self.ALLOWED_STATUS:
+            if value == self.ALLOWED_STATUS[0]:
+                self.add_child(self.titlebar)
+            elif value == self.ALLOWED_STATUS[1]:
+                self.remove_child(self.titlebar)
+        
+        self.redraw_window()
+
+    def _set_statusbar(self, value):
```

Comment:
This is almost the exact same code, just refactor both methods into a generic method you can pass `self.titlebar` to.

Reply:
Expected that feedback. I'll do :)